controller/scheduler-v2/scheduler.go

```diff
@@ -135,88 +171,102 @@ func (s *Scheduler) Sync() (err error) {
 				}
 				if _, ok := s.jobs[job.ID]; ok {
 					log.Info("skipping known job")
+					delete(inactiveJobs, job.ID)
 					continue
 				}
 
 				log.Info("getting formation")
-				f, err := s.getFormation(appID, appName, releaseID)
-				if err != nil {
-					log.Error("error getting formation", "err", err)
+				f := s.formations.Get(appID, releaseID)
+				if f == nil {
+					err = fmt.Errorf("No such formation obtained")
```

Comment:
Sure, but if we trigger the formation sync, then we have to also trigger another job sync afterwards so that we have this job in-memory, which seems like a lot of work when we could just do a lookup of the formation from the controller and move on.